### PR TITLE
The -p option must be used with the -f option

### PR DESCRIPTION
**Pull request by guy-maurel**
Ref. #930

### CODE REVIEW

**Comment by kaiser101** [comment]
[debug.txt](https://github.com/uncrustify/uncrustify/files/2098904/debug.txt)

**Comment by mwoehlke-kitware** [comment]
Does `-p` still work with this patch if I am formatting stdin?

File: src/uncrustify.cpp [hunk 1]
```diff
@@ -265,7 +265,7 @@ void usage_exit(const char *msg, const char *argv0, int code)
            " --set <option>=<value>   : Sets a new value to a config option.\n"
            "\n"
            "Debug Options:\n"
-           " -p FILE      : Dump debug info to a file.\n"
+           " -p FILE      : Dump debug info to a file. Must be use with the option -f FILE.\n"
```
Comment (mihaipopescu) [inline src/uncrustify.cpp]: use**d**

**Comment by mwoehlke-kitware** [comment]
> Does `-p` still work with this patch if I am formatting stdin?

Apparently, yes.

**Pull request merged**